packages/ui-components/src/components/Box/Box.component.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+const boxStyles = `
+  jn-text-sm
+  jn-rounded
+  jn-bg-theme-box-default
+  jn-border
+  jn-border-theme-box-default
+`
+
+// When adjusting the padding, make sure to update the tests to verify both padded and unpadded Box variations correctly.
+const boxPadding = `
+  jn-py-1
+  jn-px-2
+`
+
+export interface BoxProps {
+  /**
+   * Child elements to be rendered inside the Box.
+   */
+  children?: React.ReactNode
```

Comment:
This prop should probably be mandatory, as without content the Box component has no purpose?
There's no other props to pass content.